### ISSUE TITLE
Whining coil noise 

### ISSUE THREAD

**Issue by simonepsp** [horvathgergo/esp8266-for-fornuftig#3]
Hello everyone!
First all, thank you for this great project. I've been looking for a way to build a cheap, open source and esp based air purifier for a while and this project made it possible!

I've purchased 5 built PCBs from JLPCB and everything has been working so far.
The only disturbing issue is a constant whining coil / high pitch noise (likely) coming from the ESP32 board or the capacitors. The issue only seem to occur only when the board is powered directly with the purifier's +24V cable and not when supplied +5V.

Have you experienced / noticed anything similar? 
Thanks a lot!

![IMG_3473](https://github.com/horvathgergo/esp8266-for-fornuftig/assets/11884948/b3cac634-9a27-49b0-8421-8d23253dd822)
![Screenshot 2024-03-31 at 09 28 07@2x](https://github.com/horvathgergo/esp8266-for-fornuftig/assets/11884948/a0d087e0-20e3-485c-9d9f-5b319a16ee96)

**Comment by GreyEarl** [comment]
Yes, I can confirm I have the same. Personally I only notice the coil whine in my right ear, my left ear seems to be "older" ;)
Not sure how this can be fixed to be honest.

**Comment by simonepsp** [comment]
> Yes, I can confirm I have the same. Personally I only notice the coil whine in my right ear, my left ear seems to be "older" ;) Not sure how this can be fixed to be honest.

hahah I see... I've covered the power inductor in the middle of the PCB with some hot glue and looks like the noise is gone.
I'll keep on eye on it for any heat dissipation issue

Edit: nvm. The noise is still there. Grrr

**Comment by joaopbnogueira** [comment]
@simonepsp I'm facing the same problem but I ended up slapping 1 layer of Kilmat deadening automotive insulation mat on the bottom of the PCB and 2 layers on top making sure to get a tight seal around the components and not cover any part of the PCB used for the ESP antenna. So far so good.
